code/modules/client/preferences.dm

```diff
@@ -169,7 +169,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			dat += "<a href='?_src_=prefs;preference=name;task=random'>Random Name</A> "
 			dat += "<a href='?_src_=prefs;preference=name'>Always Random Name: [active_character.be_random_name ? "Yes" : "No"]</a><BR>"
 
-			dat += "<b>Name:</b> "
+			dat += "<b>[TOOLTIP_WRAPPER("Name:", 400, "Each speices must follow a naming policy. Please check our \"Naming Guidelines\" in our wiki.")]</b> "
```

Comment:
Please read the "Naming Guidelines" page on the wiki and Rule 1.7.

Reply:
Probably a bit too specific without this being a config entry